setup.py

```diff
@@ -32,11 +32,17 @@
     ]
 
     # use "pip install -e .[dev]" to install required components + extra components
-    extras_require = [
+    production = [
         'vivarium_cluster_tools==1.2.3',
         'vivarium_inputs[data]==4.0.2',
     ]
 
+    interactive = [
+        'jupyterlab',
+        'matplotlib',
+        'sympy',
+    ]
```

Comment:
Added this block to help install everything you might need in the environment when working interactively.